scipy/optimize/tests/test_linprog.py

```diff
@@ -2366,6 +2366,42 @@ def test_mip_rel_gap_passdown(self):
         assert not np.all(gap_diffs == 0)
 
 
```

Comment:
```suggestion
```
PEP8 only calls for 1 blank line between functions, not 2.

Reply:
The recommendation is correct here, but it's because this is a method within a class. 

![image](https://user-images.githubusercontent.com/6570539/226489905-ab31b29f-5176-481f-a2b6-97c90561af71.png)

https://peps.python.org/pep-0008/#blank-lines